google/cloud/spanner_dbapi/connection.py

```diff
@@ -124,6 +125,24 @@ def spanner_client(self):
         """
         return self._instance._client
 
+    @property
+    def schema_name(self):
+        """schema name for this database.
+
+        :rtype: str
+        :returns: "" for GoogleSQL and "public" for PostgreSQL for default schema
+        """
+        return self._schema_name
+
+    @schema_name.setter
+    def schema_name(self, value):
+        """Updates the database schema name used in the connection.
+
+        :type value: str
+        :param value: database schema name used in the connection
+        """
+        self._schema_name = value
```

Comment:
We should remove this. Setting a current schema is currently not (fully) supported, so we should also not 'pretend' that it is here.

The thing that is not supported, but that would be implicitly implied by this, is the following:

```sql
-- The following statement should automatically use "current_schema.my_table"
select *
from my_table
```